thingif/src/main/java/com/kii/thingif/command/CommandForm.java

```diff
@@ -63,21 +65,29 @@ public static Builder newBuilder() {
          * Create {@link Builder} with Actions
          * @param aliasActions list of {@link AliasAction} instances.
          * @return {@link Builder} instance.
+         * @throws IllegalArgumentException Thrown when aliasActions is null or empty.
          */
         @NonNull
         public static Builder newBuilder(
                 @NonNull List<AliasAction<? extends Action>> aliasActions) {
+            if (aliasActions == null || aliasActions.size() == 0) {
+                throw new IllegalArgumentException("aliasActions is null or empty");
+            }
             return new Builder(aliasActions);
         }
 
         /**
          * Add instance of {@link AliasAction} to action list.
          * @param aliasAction Instance of AliasAction
          * @return {@link Builder} instance
+         * @throws IllegalArgumentException Thrown when aliasAction is null.
          */
         @NonNull
         public Builder addAliasAction(
-                @NonNull AliasAction<? extends Action> aliasAction) {
+                @Nullable AliasAction<? extends Action> aliasAction) {
+            if (aliasAction == null) {
+                throw new IllegalArgumentException("aliasAction is null");
+            }
```

Comment:
I think `aliasAction` should be `@NonNull` or remove `IllegalArgumentException`.